README.md

```diff
@@ -170,3 +165,17 @@ http://localhost/frontend
 and geoserver at:
 
 http://localhost/geoserver
+
+
+## How to update the SCIP demo instance
+
+This application has not yet had a full release, and has no production instances, but there is a semipermanent demo instance running on docker-dev02. You will not need to worry about CORS or proxies when working with this instance, as a proxy is already set up for it.
+
+1. Clone this repository to anywhere on docker-dev02 and check out whatever branch you want to demo 
+2. Build the docker image with `docker build -t scip:your-branch-name -f docker/Dockerfile .`
```

Comment:
Ah, this is because there is no GitHub workflow to build Docker images yet. Pretty soon you'll want to do that, and then it will just be a matter of pulling the image from dockerhub.